pkg/kubelet/kuberuntime/security_context.go

```diff
@@ -37,9 +37,16 @@ func (m *kubeGenericRuntimeManager) determineEffectiveSecurityContext(pod *v1.Po
 
 	// TODO: Deprecated, remove after we switch to Seccomp field
```

Comment:
@saschagrunert another instance of this TODO (see https://github.com/kubernetes/kubernetes/pull/115898)

Reply:
Ugh, exactly this can be removed.

Reply:
actually may not have done the right thing - was the TODO to remove the code below the message as well from lines 38-48?

Reply:
yes, I think the code meant to be removed. I'd advice to do it in separate PR though. Easier for cherry-picks as we only want the bug fix to be cherry-picked, and this TODO addressed in 1.28+

Reply:
unless @saschagrunert you see a reason to do it in 1.27 and below

Reply:
created: https://github.com/kubernetes/kubernetes/issues/117045